sirmordred/task_panels.py

```diff
@@ -419,6 +436,9 @@ def __init__(self, conf):
                 logger.error(ex)
                 raise
 
+        if self.conf['panels'][GITHUB_REPOS]:
```

Comment:
This code is not enough to make it work it. You should add an entry in line `471` with the name of the source. Otherwise, it won't add GitHub repos entry to the menu.

Reply:
when rebased, probably that line was removed